package.json

```diff
@@ -192,7 +192,7 @@
     "tslint-no-toplevel-property-access": "0.0.2",
     "tslint-no-unused-expression-chai": "0.0.3",
     "typedoc": "^0.17.8",
-    "typescript": "~4.2.2",
+    "typescript": "^4.3.2",
```

Comment:
Since 4.2 is our minimum supported version, I don't think we can/should update this. While it's certainly important to update it while fixing this issue, and I'm glad CI caught it, I wouldn't want to miss a regression in 4.2 we could cause later.

Reply:
Also, I suspect the api_guardian issues will go away if you revert this.